hazelcast/src/test/java/com/hazelcast/internal/util/phonehome/PhoneHomeTest.java

```diff
@@ -612,6 +612,13 @@ public void testReplicatedMapMemoryCost() {
         assertThat(twoMapsMemoryCost).isGreaterThan(oneMapMemoryCost);
     }
 
+    @Test
+    public void testViridianPhoneHomes() {
+        Map<String, String> parameters;
```

Comment:
can you add a positive test case with `true` value?

Reply:
@promanenko  positive test case handled in the integration tests.